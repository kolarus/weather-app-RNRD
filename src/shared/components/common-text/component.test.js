import React from 'react';
import {render} from '@testing-library/react-native';

import CommonText from './component';

test('CommonText should match snapshot', () => {
  const {toJSON} = render(<CommonText>Test</CommonText>);

  expect(toJSON()).toMatchSnapshot();
});

test('CommonText should render passed text', () => {
  const {getByText} = render(<CommonText>Test</CommonText>);

  expect(getByText('Test')).not.toBeNull();
});

test('CommonText should render passed text with correct font size', () => {
  const {getByText} = render(<CommonText fontSize={23}>Test</CommonText>);

  expect(getByText('Test')).toHaveStyle({fontSize: 23});
});
