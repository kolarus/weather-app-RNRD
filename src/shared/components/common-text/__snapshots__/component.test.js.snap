// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonText should match snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "white",
        "fontFamily": "PT Sans",
        "fontSize": 15,
      },
      undefined,
      Object {},
    ]
  }
>
  Test
</Text>
`;
